lib/clamp-hsatools.in

```diff
@@ -39,6 +39,7 @@ HLC_OPT=$BINDIR/../../hlc/bin/opt
 HLC_LLC=$BINDIR/../../hlc/bin/llc
 HLC_ASM=$BINDIR/../../HSAILasm/HSAILasm
 NEWLIB=$BINDIR/../../lib
+ROCM_LIB=/opt/rocm/lib
```

Comment:
@nhaustov I'm a bit conservative about using absolute path here. We can't rule out the possibility there may be cases where we need to use different versions of ROCm Device Lib (ex: versions tailored for specific architecture). Would you consider make ROCm Device Lib a git submodule of hcc?